[2025-02-25T22:47:25.836+0530] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_processing_dag.process_input_file manual__2025-02-25T17:07:09.667220+00:00 [queued]>
[2025-02-25T22:47:25.839+0530] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_processing_dag.process_input_file manual__2025-02-25T17:07:09.667220+00:00 [queued]>
[2025-02-25T22:47:25.839+0530] {taskinstance.py:1361} INFO - Starting attempt 3 of 4
[2025-02-25T22:47:25.844+0530] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): process_input_file> on 2025-02-25 17:07:09.667220+00:00
[2025-02-25T22:47:25.848+0530] {standard_task_runner.py:57} INFO - Started process 90097 to run task
[2025-02-25T22:47:25.851+0530] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'fraud_detection_processing_dag', 'process_input_file', 'manual__2025-02-25T17:07:09.667220+00:00', '--job-id', '400', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/var/folders/kz/_l5jq2xx6db29k_3xyv2j8n40000gp/T/tmp6fy4tpom']
[2025-02-25T22:47:25.854+0530] {standard_task_runner.py:85} INFO - Job 400: Subtask process_input_file
[2025-02-25T22:47:25.876+0530] {task_command.py:416} INFO - Running <TaskInstance: fraud_detection_processing_dag.process_input_file manual__2025-02-25T17:07:09.667220+00:00 [running]> on host ninja-its-macbook-air-90.local
[2025-02-25T22:47:25.899+0530] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fraud_detection_processing_dag' AIRFLOW_CTX_TASK_ID='process_input_file' AIRFLOW_CTX_EXECUTION_DATE='2025-02-25T17:07:09.667220+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-25T17:07:09.667220+00:00'
[2025-02-25T22:47:25.900+0530] {data_pipeline_dag.py:55} INFO - Received kwargs: {'conf': <airflow.configuration.AirflowConfigParser object at 0x1031ee9d0>, 'dag': <DAG: fraud_detection_processing_dag>, 'dag_run': <DagRun fraud_detection_processing_dag @ 2025-02-25 17:07:09.667220+00:00: manual__2025-02-25T17:07:09.667220+00:00, state:running, queued_at: 2025-02-25 17:07:09.680486+00:00. externally triggered: True>, 'data_interval_end': DateTime(2025, 2, 25, 17, 7, 9, 667220, tzinfo=Timezone('UTC')), 'data_interval_start': DateTime(2025, 2, 25, 17, 7, 9, 667220, tzinfo=Timezone('UTC')), 'ds': '2025-02-25', 'ds_nodash': '20250225', 'execution_date': <Proxy at 0x161ea7ac0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x1620858b0>, 'execution_date', DateTime(2025, 2, 25, 17, 7, 9, 667220, tzinfo=Timezone('UTC')))>, 'expanded_ti_count': None, 'inlets': [], 'logical_date': DateTime(2025, 2, 25, 17, 7, 9, 667220, tzinfo=Timezone('UTC')), 'macros': <module 'airflow.macros' from '/Users/nc25577_suryateja/Documents/air_pro/venv/lib/python3.9/site-packages/airflow/macros/__init__.py'>, 'next_ds': <Proxy at 0x161edc900 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x1620858b0>, 'next_ds', '2025-02-25')>, 'next_ds_nodash': <Proxy at 0x15d35d900 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x1620858b0>, 'next_ds_nodash', '20250225')>, 'next_execution_date': <Proxy at 0x15d35d380 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x1620858b0>, 'next_execution_date', DateTime(2025, 2, 25, 17, 7, 9, 667220, tzinfo=Timezone('UTC')))>, 'outlets': [], 'params': {}, 'prev_data_interval_start_success': DateTime(2025, 2, 25, 16, 13, 9, 916812, tzinfo=Timezone('UTC')), 'prev_data_interval_end_success': DateTime(2025, 2, 25, 16, 13, 9, 916812, tzinfo=Timezone('UTC')), 'prev_ds': <Proxy at 0x15d35dc40 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x1620858b0>, 'prev_ds', '2025-02-25')>, 'prev_ds_nodash': <Proxy at 0x15d35da40 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x1620858b0>, 'prev_ds_nodash', '20250225')>, 'prev_execution_date': <Proxy at 0x15d35d9c0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x1620858b0>, 'prev_execution_date', DateTime(2025, 2, 25, 17, 7, 9, 667220, tzinfo=Timezone('UTC')))>, 'prev_execution_date_success': <Proxy at 0x15d35d980 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x1620858b0>, 'prev_execution_date_success', DateTime(2025, 2, 25, 16, 13, 9, 916812, tzinfo=Timezone('UTC')))>, 'prev_start_date_success': DateTime(2025, 2, 25, 16, 13, 11, 106050, tzinfo=Timezone('UTC')), 'run_id': 'manual__2025-02-25T17:07:09.667220+00:00', 'task': <Task(PythonOperator): process_input_file>, 'task_instance': <TaskInstance: fraud_detection_processing_dag.process_input_file manual__2025-02-25T17:07:09.667220+00:00 [running]>, 'task_instance_key_str': 'fraud_detection_processing_dag__process_input_file__20250225', 'test_mode': False, 'ti': <TaskInstance: fraud_detection_processing_dag.process_input_file manual__2025-02-25T17:07:09.667220+00:00 [running]>, 'tomorrow_ds': <Proxy at 0x15d35dd00 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x1620858b0>, 'tomorrow_ds', '2025-02-26')>, 'tomorrow_ds_nodash': <Proxy at 0x15d35dd40 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x1620858b0>, 'tomorrow_ds_nodash', '20250226')>, 'triggering_dataset_events': <Proxy at 0x1620902c0 with factory <function TaskInstance.get_template_context.<locals>.get_triggering_events at 0x162085670>>, 'ts': '2025-02-25T17:07:09.667220+00:00', 'ts_nodash': '20250225T170709', 'ts_nodash_with_tz': '20250225T170709.667220+0000', 'var': {'json': None, 'value': None}, 'conn': None, 'yesterday_ds': <Proxy at 0x15d35dac0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x1620858b0>, 'yesterday_ds', '2025-02-24')>, 'yesterday_ds_nodash': <Proxy at 0x15d35db00 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x1620858b0>, 'yesterday_ds_nodash', '20250224')>, 'templates_dict': None}
[2025-02-25T22:47:25.900+0530] {data_pipeline_dag.py:59} INFO - Conf type: <class 'airflow.configuration.AirflowConfigParser'>, value: <airflow.configuration.AirflowConfigParser object at 0x1031ee9d0>
[2025-02-25T22:47:25.900+0530] {data_pipeline_dag.py:68} WARNING - No file_path in conf; defaulting to /Users/nc25577_suryateja/Documents/air_pro/input_files/uploaded_file
[2025-02-25T22:47:25.901+0530] {data_pipeline_dag.py:70} INFO - Processing input file at /Users/nc25577_suryateja/Documents/air_pro/input_files/uploaded_file
[2025-02-25T22:47:25.901+0530] {data_pipeline_dag.py:73} ERROR - File not found: /Users/nc25577_suryateja/Documents/air_pro/input_files/uploaded_file
[2025-02-25T22:47:25.901+0530] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/Users/nc25577_suryateja/Documents/air_pro/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/Users/nc25577_suryateja/Documents/air_pro/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/Users/nc25577_suryateja/Documents/air_pro/airflow/dags/data_pipeline_dag.py", line 74, in process_input_file
    raise FileNotFoundError(f"File not found: {file_path}")
FileNotFoundError: File not found: /Users/nc25577_suryateja/Documents/air_pro/input_files/uploaded_file
[2025-02-25T22:47:25.917+0530] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=fraud_detection_processing_dag, task_id=process_input_file, execution_date=20250225T170709, start_date=20250225T171725, end_date=20250225T171725
[2025-02-25T22:47:25.920+0530] {standard_task_runner.py:104} ERROR - Failed to execute job 400 for task process_input_file (File not found: /Users/nc25577_suryateja/Documents/air_pro/input_files/uploaded_file; 90097)
[2025-02-25T22:47:25.946+0530] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-02-25T22:47:25.956+0530] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
